src/main/java/org/jitsi/protocol/xmpp/util/MediaSSRCGroupMap.java

```diff
@@ -33,22 +34,22 @@
     /**
      * Map backend.
      */
-    private final Map<String, List<SSRCGroup>> groupMap;
+    private final ConcurrentHashMap<String, List<SSRCGroup>> groupMap;
```

Comment:
What's the purpose of this? The whole MediaSSRCGroupMap is thread-unsafe so is this single change supposed to make it thread-safe? I don't think it does but then the purpose that you had in mind may be different.